qualtran/_infra/data_types.py

```diff
@@ -530,6 +530,23 @@ class QFxp(QDType):
     We can specify a fixed point real number by the tuple bitsize, num_frac and
     signed, with num_int determined as `(bitsize - num_frac)`.
 
+
+    Classical Simulation:
```

Comment:
doc nit: These "numpydoc-style" headings have semantic meaning. Otherwise, use markdown. Here, I'd suggest using a `<h3>`

```
### Classical simulation

To hook into the classical simulator...
```

Reply:
fixed